### PR TITLE
Use filter+count instead?

### PR DESCRIPTION
**Pull request by Kimbsy**
My Java is super rusty, but would this work?

Apparently I'm still on the Trainline leaderboard :grin:

### CODE REVIEW

**Comment by menasameh** [comment]
Yeah, it works perfectly!
but the return type from count would be `long` instead of `int`, 
I was thinking of it like a filter but at 5 am that's how a filter would look in my head 🤣

File: src/day4/Day4.java [hunk 1]
```diff
@@ -13,9 +13,8 @@ public void smallSol() {
         List<List<Range>> list = getInput();
 
         int ans = list.stream()
```
Comment (menasameh) [inline src/day4/Day4.java]:
```suggestion
        long ans = list.stream()
```

File: src/day4/Day4.java [hunk 2]
```diff
@@ -24,9 +23,8 @@ public void largeSol() {
...
```
Comment (menasameh) [inline src/day4/Day4.java]:
```suggestion
        long ans = list.stream()
```

**Comment by Kimbsy** [comment]
Haha yeah I'm not doing the 5am starts this year. I really enjoyed pushing myself last year, but I'm just going for casual this time round ☺️

Good luck on the leaderboard!

**Pull request merged**